hppc/src/test/templates/com/carrotsearch/hppc/SortedIterationKTypeVTypeHashMapTest.java

```diff
@@ -26,19 +31,24 @@
   @Before
   public void initDelegate() {
     hasEmptyKey = randomBoolean();
-    sortedKeys = hasEmptyKey ? newArray(keyE, key1, key2, key3, key4) : newArray(key1, key2, key3, key4);
-    sortedValues = hasEmptyKey ? newvArray(value0, value1, value2, value3, value4) : newvArray(value1, value2, value3, value4);
+    sortedKeys =
+        hasEmptyKey ? newArray(keyE, key1, key2, key3, key4) : newArray(key1, key2, key3, key4);
+    sortedValues =
+        hasEmptyKey
+            ? newvArray(value0, value1, value2, value3, value4)
+            : newvArray(value1, value2, value3, value4);
     for (int i = sortedKeys.length - 1; i >= 0; i--) {
       delegate.put(sortedKeys[i], sortedValues[i]);
     }
   }
 
   @Test
   public void testGetContains() {
-    SortedIterationKTypeVTypeHashMap<KType, VType> view = new SortedIterationKTypeVTypeHashMap<KType, VType>(delegate);
+    SortedIterationKTypeVTypeHashMap<KType, VType> view =
+        new SortedIterationKTypeVTypeHashMap<KType, VType>(delegate, getNaturalComparator());
```

Comment:
I miss the constructor without comparator for simpler code.
Could we have it, this time just a sugar that calls the constructor with comparator and using KTypeComparator#naturalOrder (see my comment below)?

Reply:
I removed it intentionally so that it's explicit how the order is specified, actually. You can pass a comparator method reference (Comparable::compare, if you like) and this is really compact. At the same time, there are no hidden requirements about the "natural" order (like KType implementing Comparable) - this is the bit that didn't really fit well to me.